float32 acceleration
---
bool result
